specs/ledger/hs/src/Ledger/Core.hs

```diff
@@ -83,17 +83,17 @@ verify (VKey vk) vd (Sig sd sk) = vk == sk && vd == sd
 -- Slots and Epochs
 ---------------------------------------------------------------------------------
 
-newtype Epoch = Epoch Natural
+newtype Epoch = Epoch Word64
   deriving (Eq, Ord, Show, HasTypeReps)
 
-newtype Slot = Slot Natural
+newtype Slot = Slot { unSlot :: Word64 }
   deriving (Eq, Ord, Show, HasTypeReps)
 
 -- | A number of slots.
 --
 --   We use this newtype to distinguish between a cardinal slot and a relative
 --   period of slots.
-newtype SlotCount = SlotCount Natural
+newtype SlotCount = SlotCount Word64
```

Comment:
we use `Duration` in the new ledger spec. Would it make sense to use it here, too?

Reply:
Definitely. @nc6, any objections? (damn, I feel so holacratic today ...)

Reply:
No objections

Reply:
I'm thinking I'm not happy with `Duration` either. I'd prefer `SlotDuration`, but then `SlotCount` conveys a more precise meaning ... I think I'll leave this as is for the time being, and we can discuss later what we want.